File: src/main/java/com/reddot/ReddotApplication.java
package com.reddot;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;

@SpringBootApplication
@EnableAsync
public class ReddotApplication {

	public static void main(String[] args) {
		SpringApplication.run(ReddotApplication.class, args);
	}

}

File: src/main/kotlin/com/reddot/ReddotApplication.kt
package com.reddot

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableAsync

@SpringBootApplication
@EnableAsync
class ReddotApplication

fun main(args: Array<String>) {
	runApplication<ReddotApplication>(*args)
}
